scripts/protoc.sh

```diff
@@ -24,17 +24,16 @@ mkdir -p $OUT_DIR || true
 cd "${ROOT_DIR}"/protocol || exit
 
 PROTOC_GEN_TS_PATH="${ROOT_DIR}/node_modules/.bin/protoc-gen-ts"
-PROTOC_PLUGIN="${ROOT_DIR}/node_modules/.bin/grpc_tools_node_protoc_plugin"
+#PROTOC_PLUGIN="${ROOT_DIR}/node_modules/.bin/grpc_tools_node_protoc_plugin"
```

Comment:
Why comment out? Rather than deleting?

Reply:
new grpc not use grpc_tools_node_protoc_plugin.
https://www.npmjs.com/package/grpc_tools_node_protoc_ts

Reply:
This is not the point. The key is removing useless codes, not comment them out.

Reply:
ok